frontend/src/scenes/cohorts/CohortEdit.tsx

```diff
@@ -280,8 +280,9 @@ export function CohortEdit({ id }: CohortLogicProps): JSX.Element {
                                 Persons in this cohort
                                 <span className="text-secondary ml-2">
                                     {!cohort.is_calculating &&
+                                        cohort.count !== undefined &&
```

Comment:
I think we have some sort of eventual consistency issue where after an upload, `cohort.count` will be null here even though we have the count in the Postgres `posthog_cohort` table. I didn't dig into it too much, but I think it has to do with Clickhouse because if you refresh after a little bit, the count shows up.

So this change just hides the count message until we have something to show. Prior to this it would say "matching null persons" which just looks broken.